CONTRIBUTING.md

```diff
@@ -1,5 +1,8 @@
 # Contributing to cleanlab
 
+*If you're looking to contribute code to cleanlab, you may also be interested
+in the [development guide](DEVELOPMENT.md)*
```

Comment:
May want to link DEVELOPMENT.md again down in the Pull Requests section of this document:

https://github.com/cleanlab/cleanlab/blob/568fa01ccdd1685565e282abade2057f41b56217/CONTRIBUTING.md#pull-requests

In general, the pull requests section has some content that's redundant with DEVELOPMENT.md (that's ok with me though, just letting you know in case you prefer less redundancy).

Reply:
Thanks, fixed.